openhtf/io/output/print_ToConsole.py

```diff
@@ -0,0 +1,93 @@
+"""Module to display test summary on console."""
+import os
+import sys
+from openhtf import util
+from openhtf.io import test_record as tr
+from openhtf.util import measurements as mea
```

Comment:
The style guide says to not rename unless it's a long name, and we haven't renamed these yet because they're not that long. I've seen 'numpy as np' often because numpy code uses that namespace often enough, but this code doesn't justify a rename, it only makes it harder to grep for uses and know what 'mea' or 'tr' mean in the code